openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -383,10 +418,14 @@ def create_media_view(
 
 def update_db_view(
     postgres_conn_id,
+    task,
```

Comment:
The `refresh_matview` task also needs to be updated to prevent the `ValueError` here.